### PR TITLE
Transactions management

### PR DESCRIPTION
**Pull request by rostIvan**

### CODE REVIEW

File: src/main/java/com/ita/if103java/ims/dto/TransactionDto.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        this.id = id;
+        this.timestamp = timestamp;
+        this.type = type;
+        this.accountDto = accountDto;
+        this.workerId = workerId;
+        this.itemDto = itemDto;
+        this.quantity = quantity;
+        this.movedFrom = movedFrom;
+        this.movedTo = movedTo;
+    }
```
Comment (volodymyrvynnyk) [inline src/main/java/com/ita/if103java/ims/dto/TransactionDto.java]: I think it's better to make constructors like Builder design patter. I've just made it in Event entity and dto (branch event-controller)

Comment (rostIvan) [reply src/main/java/com/ita/if103java/ims/dto/TransactionDto.java]: Yep, I've checked, thanks
I'll think about that ;)

File: src/main/java/com/ita/if103java/ims/controller/TransactionController.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    private TransactionService transactionService;
+
+    @Autowired
+    public TransactionController(TransactionService transactionService) {
+        this.transactionService = transactionService;
+    }
+
+    @GetMapping("/{id}")
+    public TransactionDto findById(@PathVariable("id") BigInteger id) {
```
Comment (ikrasWork) [inline src/main/java/com/ita/if103java/ims/controller/TransactionController.java]: hm, Integer or Long will be enough

File: src/main/java/com/ita/if103java/ims/service/impl/TransactionServiceImpl.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        // TransactionType -> (IN, OUT)
+        return new TransactionDto(
+            transaction.getId(),
+            transaction.getTimestamp(),
+            transaction.getType(),
+            accountService.view(transaction.getAccountId()),
+            userService.findById(transaction.getWorkerId()),
+            itemService.findById(transaction.getItemId()),
+            transaction.getQuantity(),
+            associateService.view(transaction.getAssociateId())
```
Comment (ikrasWork) [inline src/main/java/com/ita/if103java/ims/service/impl/TransactionServiceImpl.java]: hm, So in this case we do not have information from what warehouse we sold items, or to what warehouse we put items?
We need this info.

Comment (rostIvan) [reply src/main/java/com/ita/if103java/ims/service/impl/TransactionServiceImpl.java]: Got it, will fix asap

**Pull request merged**